packages/node/test/unit/json-fixture/README.md

```diff
@@ -63,14 +63,6 @@ Now that you have a snapshot, you can use it in a test case:
 >
 > Commit `project.json`, but not the `project` directory. See the [Modifying a Fixture][] section for why not!
 
-### Fixtures with Binary Data and/or Symlinks
-
-If you need to create a fixture containing binary data (_use case: native modules_) or symlinks (_use case: `node_modules/.bin/*`_) provide the `--binary` flag to [snapshot-fs][]:
-
-```sh
-npx snapshot-fs project.json --dir project --binary
-```
-
 ## Modifying a Fixture
 
 > New in [`snapshot-fs`][snapshot-fs] v3.1.0
```

Comment:
This section seems outdated.

Reply:
I don't think so... I think you may have been looking at the wrong section.

_This_ section shows how to _create a directory structure from a snapshot_. What you referred to in the other comment in the test file sounded more like you wanted to _create a snapshot from a directory structure_.

The title is misleading, though. I'll change that.

Reply:
- [x] changed